src/views/ApplyPage/components/FileInput/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useRef, useState } from 'react';
+
+import IconPlusButton from './icons/IconPlusButton';
+import { container, errorText, fileInput, fileLabelVar, fileNameVar, textWrapper } from './style.css';
+
+const FileInput = ({ disabled }: { disabled?: boolean }) => {
+  const [isError, setIsError] = useState(false);
+  const [file, setFile] = useState<File | null>(null);
+  const fileName = file ? file.name : '';
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const handleChangeFile = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    const LIMIT_SIZE = 1024 ** 2 * 500; //50KB
+    if (file) {
+      if (LIMIT_SIZE < file.size) {
+        setIsError(true);
+      } else {
+        setIsError(false);
+        setFile(file);
+      }
```

Comment:
파일을 업로드 했는데 다른 파일로 변경하고 싶어서 새로운 파일을 업로드 하려고 하는데, 만약 새로운 파일이 50mb 넘어갈 시 아래와 같이 됩니다
<img width="786" alt="스크린샷 2024-06-25 오후 8 34 50" src="https://github.com/sopt-makers/sopt-recruiting-frontend/assets/121864459/3b1f0e33-599e-45ca-860f-4fb0cbb6d24f">
기존의 파일은 유지되면서 에러 메세지가 되는 거죠!

근데 이 상태로 제출 시 파일이 업로드 되어 있는 걸로 되어서 에러 메세지는 뜨지만 제출은 되는 경우가 발생할 거 같아요!

그래서 에러가 뜨면 기존 파일도 제거해주는 방식으로 가는 거 어떤가요?? 어차피 파일 수정하려고 재업로드 하는 거라 기존 파일은 사용 안 할 거였다는 거니까요!

```suggestion
    if (file) {
      if (LIMIT_SIZE < file.size) {
        setIsError(true);
        if (inputRef.current) {
          inputRef.current.value = '';
          setFile(null);
        }
      } else {
        setIsError(false);
        setFile(file);
      }
```

Reply:
헉!! 그렇네요 버그 잡아주셔서 감사합니다 🐛🐛🐛🐛🐛